tests/testthat/test-ggplot-heatmap.R

```diff
@@ -4,10 +4,12 @@ wdays <- c("Monday", "Tuesday", "Wednesday", "Thursday", "Friday")
 dtimes <- c("Morning", "Afternoon", "Evening")
 workweek <- matrix(c(1, 20, 30, 20, 1, 60, 30, 60, 1, 50, 80, -10, 1, 30, 20),
              nrow=5, ncol=3, byrow=TRUE,
-             dimnames=list(wdays, dtimes))
+             dimnames=list(day=wdays, time=dtimes))
 ww <- reshape2::melt(workweek)
+ww$day <- factor(ww$day, wdays)
```

Comment:
Did you add this just to make it explicit, and/or to handle some possible subsetting?
Because `ww$day` is already a factor...
These tests are hard-coded so I'm not sure that's necessary.